plugins/eza/eza.plugin.zsh

```diff
@@ -0,0 +1,46 @@
+if ! (( $+commands[eza] )); then
+  print "zsh eza plugin: eza not found. Please install eza before using this plugin." >&2
+  return 1
+fi
+
+typeset -a _EZA_HEAD
+typeset -a _EZA_TAIL
+
+function _configure_eza() {
+  local _val
+  # Get the head flags
+  zstyle -s ':omz:plugins:eza' 'showgroup' _val
+  if [[ -z $_val || $_val == 1 ]]; then
+    _EZA_HEAD+=("g")
+  fi
+  # Get the tail long-options
+  zstyle -s ':omz:plugins:eza' 'dirs-first' _val
+  if [[ $_val == 1 ]]; then
+    _EZA_TAIL+=("--group-directories-first")
+  fi
+  zstyle -s ':omz:plugins:eza' 'time-style' _val
+  if [[ $_val ]]; then
+    _EZA_TAIL+=("--time-style='$_val'")
+  fi
+}
```

Comment:
I'd instead use the same way of `zstyle` as in https://github.com/ohmyzsh/ohmyzsh/blob/c891b7e6ada5427f4c8cca05c8ed1ae33cb49943/plugins/yarn/yarn.plugin.zsh#L1-L11.
With `-T` is truthy by default, and `-t` is falsy by default.

Reply:
Ahh good idea! Never knew about the `-T`|`-t` flag before!

Reply:
Implemented in commit [c7c7dd8](https://github.com/ohmyzsh/ohmyzsh/pull/12237/commits/c7c7dd81ac8cb7366c34946d980d104474adc397)